oneSpotCrackSearch/crackCheck.py

```diff
@@ -1,6 +1,27 @@
 import requests
 import json
+import urllib
+from time import sleep
+import sys
 
-game_link = "https://api.crackwatch.com/api/games?page=1"
-response = requests.get(game_link)
-print(response.txt)
\ No newline at end of file
+def Crack_checker(link,title):
+    url = urllib.request.urlopen(link)
+    data = json.loads(url.read())
+    strdata = str(data)
+    if(strdata!="[]"):
+        if (strdata.find(title) != -1):
+            return True
+        else:
+            return False
+    else:
+        print("Game is not cracked")
+        sleep(20)
+        sys.exit(0)
```

Comment:
The code works, if the game is cracked. There does seem to be a problem when it comes to reporting if a game is not cracked though, I'm assuming that it has something to do with Line 24, inside the url, ```is_cracked=true```.

However, it can be improved on later, nice work!

Reply:
Yes, Since the purpose of the app was to check if the games are cracked or not. I directly searched only for the cracked ones. If the game is not present in all the various pages provided, then the obvious conclusion would be that the game isn't cracked. It would be futile to search through the entire database just to realise that the game isn't cracked.

Reply:
_**Indeed**_
However, the exceptions that were received were the problem, not the code for the checking.
You've seemed to have resolved them with #5's try and except, so, it's no longer much of a problem.